scripts/generate_dump

```diff
@@ -2201,8 +2213,24 @@ remove_secret_from_etc_files() {
     sed -i -E 's/(\s*snmp_\S*community\s*:\s*)(\S*)/\1****/g' $dumppath/etc/sonic/snmp.yml
 
     # Remove secret from /etc/sonic/config_db.json
-    cat $dumppath/etc/sonic/config_db.json | remove_secret_from_config_db_dump > $dumppath/etc/sonic/config_db.json.temp
-    mv $dumppath/etc/sonic/config_db.json.temp $dumppath/etc/sonic/config_db.json
+    remove_secret_from_config_db_dump_file $dumppath/etc/sonic/config_db.json
+
+    # Remove secret from /etc/sonic/golden_config_db.json
+    remove_secret_from_config_db_dump_file $dumppath/etc/sonic/golden_config_db.json
+
+    # Remove secret from /etc/sonic/old_config/
```

Comment:
No actual work here?

Reply:
No, this comments is for next 3 lines of code: 

# Remove snmp community string from old_config/snmp.yml
# Remove secret from /etc/sonic/config_db.json
# Remove secret from /etc/sonic/golden_config_db.json